### ISSUE TITLE
G3000: VOR duplicity in flight plan cause problems 

### ISSUE THREAD

**Issue by kutak11** [Working-Title-MSFS-Mods/fspackages#566]
Sometimes flight plan input stucks on VOR (3 letters) when duplicity occurs. When I choose the nearest VOR this is not added to flightplan and there is no more possible to add any further waypoint beacuase this 3 letter point stucks in "add waypoint field" ... restart of flight is necessary

**Comment by PM3Delta** [comment]
Yes I can confirm this... I just selected the FAWU1B STAR into EDDF.  It uses the VOR CHA (Charlie) a bit southeast of EDDF airport.  However, the onboard flightplanner gets it confused with another VOR some 5000 miles away, so you get a very long leg all over sudden between SPESA and DF606.  There is a trick though, somehow unloading and reloading fixed it, but yes the flightplanner gets confused.

**Comment by e-racer** [comment]
Do you have Navigraph Beta Version installed? If yes try uninstalling it and try ist again.

Best Regards
